composer.json

```diff
@@ -3,7 +3,7 @@
     "description": "Enabling cross-origin resource sharing (CORS) requests to Magento 2 API from configured Origin domain",
     "homepage": "https://github.com/splashlab/magento-2-cors-requests",
     "type": "magento2-module",
-    "version": "100.0.3",
+    "version": "100.3.3",
```

Comment:
Should be 100.0.4, as this is a minor change, isn't it ?

Reply:
Yeah I'm going to do 100.0.4, thanks